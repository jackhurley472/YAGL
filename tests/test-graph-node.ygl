Graph G;
Node n("Adam");
Node n2("James");
Node n3("Tank");
Node n4("Jack");
G + n;
G + n2;
G + n3;
G + n4;
printGraph(G);
