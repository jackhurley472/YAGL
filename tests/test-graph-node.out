============== Graph Print ===============
	Stats: 	4	4	0

Nodes:
Node (0) : Adam --- Node (1) : James --- Node (2) : Tank
Node (3) : Jack

Edges:
There aren't any edges in this graph.
============ End Graph Print =============
